blocks/gnav/gnav.css

```diff
@@ -1,389 +1,538 @@
+  display: none;
+  align-items: center;
 }
 
-header .gnav .gnav-section .gnav-selected {
-  font-weight: 700;
-  box-sizing: border-box;
+.gnav.is-Open .gnav-search-bar {
+  display: block;
 }
 
-header .gnav .gnav-section .gnav-selected > div {
-  font-weight: 400;
+.gnav-search button svg {
+  width: 20px;
+  height: 20px;
 }
 
-header .gnav .gnav-button {
-  font-size: 14px;
-  margin-left: 0px;
-  font-weight: 700;
-  padding: 6px 16px 5px 16px;
-  background-color: #1473e6;
-}      
+.gnav-search .gnav-search-button:hover {
+  fill: #2b9af3;
+}
 
-header .gnav .gnav-selected {
-  font-weight: 700;
+.gnav-search.is-Open .gnav-search-button:after {
+  content: "\2715";
+  width: 20px;
+  /* line-height: 1; */
+  font-size: 24px;
+  padding: 1px 0 0 0;
 }
 
-header .gnav-search-icon > svg {
+.gnav-search.is-Open .gnav-search-button svg {
   display: none;
 }
 
-header .gnav .gnav-nosearch .gnav-search-results {
+.gnav-search-bar {
   display: none;
+  background: #FFF;
+  border-bottom: 1px solid #EAEAEA;
+  padding-bottom: 20px;
 }
 
-header .gnav .gnav-nosearch .gnav-search-link {
-  display: none;
+.gnav-curtain.is-Open {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background: rgba(0,0,0,0.75);
+  backdrop-filter: blur(1em);
 }
 
+.gnav-search-field {
+  position: relative;
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: 1fr auto;
+  grid-gap: 20px;
+  padding: 0 20px;
+  box-sizing: border-box;
+}
 
-header .gnav-search-box {
-  max-width: 1200px;
-  margin: auto;
+.gnav-search-field svg {
+  width: 20px;
+  height: 20px;
+  position: absolute;
+  top: 6px;
+  left: 26px;
+  fill: rgb(110, 110, 110);
+  transition: fill 130ms ease-in-out;
 }
 
-header .gnav-search-box-wrapper {
-  position: relative;
-  padding: 20px;
+.gnav-search-field:hover svg {
+  fill: rgb(44, 44, 44);
+}
+
+.gnav-search-field:hover .gnav-search-input {
+  border-color: rgb(179, 179, 179);
+}
+
+.gnav-search-field .gnav-search-input {
+  display: block;
   width: 100%;
+  line-height: 30px;
+  border: 1px solid;
+  border-color: rgb(202, 202, 202);
+  border-radius: 4px;
+  padding: 0 30px;
+  font-size: 14px;
+  font-family: 'adobe-clean';
+  outline: none !important;
+  -webkit-appearance: none;
   box-sizing: border-box;
-  background-color: white;
+  transition: border-color 130ms ease-in-out, box-shadow 130ms ease-in-out;
 }
 
-header .gnav-search-results {
-  margin: 16px 0 0 0;
-  display: flex;
-  flex-wrap: wrap;
-  box-sizing: border-box;
-  width: 100%;
+.gnav-search-field a {
+  line-height: 32px;
+  background: rgb(20, 115, 230);
+  color: white;
+  border-radius: 16px;
+  font-size: 14px;
+  font-weight: 600;
+  padding: 0 16px;
 }
 
-header .gnav-search-results, header .gnav-search-link {
-  background-color: #f4f4f4;  
-  padding: 20px;
-  width: 100%;
+.gnav-search-field .gnav-search-input:focus {
+  border-color: rgb(42, 124, 223);
+}
+
+.gnav-search-input::placeholder {
+  font-style: italic;
```

Comment:
`italic` is not used

Reply:
https://opensource.adobe.com/spectrum-css/search.html Yes it is. :-)